spec/src/main/asciidoc/architecture.adoc

```diff
@@ -182,15 +182,19 @@ to `true` and have multiple methods annotated as metric that have the same metho
 .Example of reused counters
 [source,java]
 ----
-    @Counted(name = "countMe", absolute = true, reusable = true)
+    @HitCounted(name = "countMe", absolute = true, reusable = true)
```

Comment:
would be good to leave examples of `@Counted` as they are and add new examples for the new annotations.